src/Microsoft.OData.Core/Json/JsonReader.cs

```diff
@@ -22,7 +22,11 @@ namespace Microsoft.OData.Json
     /// Reader for the JSON format. http://www.json.org
     /// </summary>
     [DebuggerDisplay("{NodeType}: {Value}")]

```

Comment:
Check if `DebuggerDisplay` attribute can accommodate the `GetValue` method and if it cannot remove the `Value` property since it no longer exists

Reply:
Using `GetValue()` works as expected. Let me update the PR:

![image](https://github.com/OData/odata.net/assets/8460169/51f5f288-30ab-48cf-a2a2-f209b1102a6f)